sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -246,304 +247,620 @@ abstract class RapidsShuffleThreadedWriterBase[K, V](
     numWriterThreads: Int)
       extends RapidsShuffleWriter[K, V]
         with RapidsShuffleWriterShimHelper {
-  private val metrics = handle.metrics
-  private val serializationTimeMetric =
-    metrics.get(METRIC_SHUFFLE_SERIALIZATION_TIME)
-  private val shuffleWriteTimeMetric =
-    metrics.get(METRIC_SHUFFLE_WRITE_TIME)
-  private val shuffleCombineTimeMetric =
-    metrics.get(METRIC_SHUFFLE_COMBINE_TIME)
-  private val ioTimeMetric =
-    metrics.get(METRIC_SHUFFLE_WRITE_IO_TIME)
   private val dep: ShuffleDependency[K, V, V] = handle.dependency
   private val shuffleId = dep.shuffleId
   private val partitioner = dep.partitioner
   private val numPartitions = partitioner.numPartitions
   private val serializer = dep.serializer.newInstance()
   private val transferToEnabled = sparkConf.getBoolean("spark.file.transferTo", true)
-  private val fileBufferSize = sparkConf.get(config.SHUFFLE_FILE_BUFFER_SIZE).toInt * 1024
+  private val fileBufferSize = 64 << 10
```

Comment:
why? Anytime I see a magic number I want to understand how we got that number.  Also why are we replacing a config for this? If 64k is much better than 32k, okay, but should we have a separate rapids config for this instead? is 64k always the ideal?